tests/units/oup/test_oup_parser.py

```diff
@@ -313,7 +313,7 @@ def test_no_abstract_value(shared_datadir, parser):
 
 def test_titles(parsed_articles):
     titles = [
-        "Detectable electric current induced by the dark matter axion in a conductor",
+        "Detectable electric current induced by the dark matter axion in aconductor",
```

Comment:
why is that? 😄

Reply:
:DDDDD I think it was by mistake, I deleted the space in the XML and then "mm, tests are failing  "

Reply:
mmm actually is after the file formatting

Reply:
fixed: actually, I needed to change the cleaning string value, after parsing (in xml_extractors text_extractor)